homeassistant/components/satel_integra/__init__.py

```diff
@@ -82,6 +110,8 @@
     if not result:
         return False
 
+    conf[CONTROLLER] = controller
```

Comment:
We can't pass the controller in the discovery_info. Only serializable data can be passed as discovery_info. Just pick the controller up from hass.data in the platforms.

Reply:
OK, looks like I did not understood your intentions after all. WIll fix in a minute.